docs/UserGuide.md

```diff
@@ -453,46 +449,75 @@ Examples:
+
+:information_source: **Notes:**
+* `DATE TIME` and `INTERVAL TIME_PERIOD` must follow the criteria defined in [Task parameters](#parameter-constraints).
+* Note that tasks are automatically sorted in chronological order upon being added, i.e if a task on `25-10-22` is added, this task's `TASK_INDEX` would be based on its chronological order in the patients task list. E.g the patient already contains tasks on `24-10-22` and `27-10-22`, the new task will be the 2nd task for the patient after the one on `24-10-22`.
+* If a `TIME` of `2400` if provided, it would simply be translated into `0000` hours of the next day. <br>
+  e.g `2-11-22 2400` will be automatically be inferred as `3-11-22 0000`.
+* If the day portion of the date exceeds the last day of that calendar month, it would default to the last day of the month. <br>
+  e.g `31-4-22` will be automatically converted to `30-4-22` or `30-2-20` will be converted to `29-2-20` since 2020 is a leap year.
+
+</div>
 
 Examples:
 * `list` followed by `add -p 1 d/Administer 3ml of example medicine` adds a task to the 1st patient in the patient list.
 * `find Betsy` followed by `add -p 2 d/Change dressing on left arm | 12-7-22` adds a task to the 2nd patient in results of the `find` command, on 12th July 2022 0000 hours.
 * `add -p 3 d/Take X-rays | 23-4-22 1345 | 3 weeks` adds a recurring task to the 3rd patient for every 3 weeks starting from 23rd April 2022 1345 hours.
 
+<div markdown="block" class="alert alert-success">
+
+:bulb **Tip:** You can add multiple tasks at once when you first [add a patient](#adding-a-patient-add).
+
+</div>
+
 <br>
 
 ### Editing a task: `edit` `-p` `-d`
 
-Edits the specified task or recurring task associated with a patient.
+You can edit a task of a patient with the `edit` command.
+
+Format: **`edit`** `-p PATIENT_INDEX -d TASK_INDEX d/<TASK_DESCRIPTION> | <DATE TIME> | <INTERVAL TIME_PERIOD>`
 
-Format: `edit -p PATIENT_INDEX -d TASK_INDEX d/<TASK_DESCRIPTION> | <DATE TIME> | <INTERVAL TIME_PERIOD>`
+<div markdown="block" class="alert alert-info">
 
-* Edits the task at the specified `TASK_INDEX` of the patient at the specified `PATIENT_INDEX`.
-* The task index refers to the index number shown in the task list of a patient.
-* `DATE TIME` and `INTERVAL TIME_PERIOD` must follow the criteria defined in notes under command format.
-* If no new `DATE TIME` or `INTERVAL TIME_PERIOD` are provided, then original values will be used.
+:information_source: **Notes:**
+* `DATE TIME` and `INTERVAL TIME_PERIOD` must follow the criteria defined in [Task parameters](#parameter-constraints).
 * If a `INTERVAL TIME_PERIOD` is provided for what was originally a non-recurring task, the edit will transform it into a recurring one based on the given frequency
+* Tasks are automatically sorted in chronological order upon modification, i.e. if a task on `25-10-22` is edited to be `30-10-22`, its new `TASK INDEX` would be based on the displayed order in the patient's task list.
```

Comment:
Repetitive. Mention these details once the first time the user encounters it i.e. in the add task section. Advanced users who use this guide as a refresher would probably know these details.

Reply:
samme goes for the repeated parts below